src/universe/topics.ts

```diff
@@ -6,75 +6,55 @@ interface IInjectableDataObject {
 }
 
 abstract class TopicGenerator {
-  public abstract template: string
+  public abstract get template (): string
   public abstract generateTopic (data?: IInjectableDataObject, ...args: any[]): string
-  public abstract isTopic (topic: string, data?: IInjectableDataObject): boolean
+  public isTopic (topic: string, data?: IInjectableDataObject): boolean {
+    return topic === this.template
+  }
 }
 
+const uuidRegex = '[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}'
+
 export default {
   api: {
     message: new (class extends TopicGenerator {
-      template: string = 'api/message'
+      template = 'api/message'
```

Comment:
As discussed getter in the parent class is implicitly converted to a field in the child class.